microsoft-edge/devtools-guide-chromium/landing/index.yml

```diff
@@ -0,0 +1,646 @@
+#    4 cards, 3 4 6  4 articles
+
+# Azure Active Directory Identity Governance documentation: https://docs.microsoft.com/en-us/azure/active-directory/governance/
+#    5 cards, 2 4 3  1 1 articles
+
+# Azure Data Box documentation: https://docs.microsoft.com/en-us/azure/databox/
+#    6 cards, 4 4 3  3 3 6 articles
+
+# Azure AD Domain Services documentation: https://docs.microsoft.com/en-us/azure/active-directory-domain-services/
+#    6 cards, 4 3 5  5 4 4 articles
+
+# Azure IoT Edge documentation: https://docs.microsoft.com/en-us/azure/iot-edge/?view=iotedge-2020-11
+#    7 cards, 3 3 3  3 4 8  3 articles
+
+# Microsoft identity platform: https://docs.microsoft.com/en-us/azure/active-directory/develop/
+#    12 cards, 3 3 3  3 4 3  2 2 2  2 2 1 articles
+
+
+# =============================================================================
+# Guidelines about Landing page card design:
+
+# https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-landing-page?branch=master#guidelines
+# Cards and links should be based on top customer tasks or top subjects.
+# Don't just repeat the TOC.
+# Order cards by relevancy.
+# Card titles should start with a verb.
+
+# Per Research section above: 
+# IDE landing pages typically have 5-7 cards, 4-6 article-links each.
+
+
+# =============================================================================
+landingContent:
+
+# might be a suggested card title:   Device Emulation
+
+# =============================================================================
+# Card
+  - title: Getting started with DevTools
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Reasons to use Microsoft Edge DevTools
+            url: /microsoft-edge/devtools-guide-chromium/why-use-microsoft-edge-devtools
+          - text: Open DevTools
+            url: /microsoft-edge/devtools-guide-chromium/open/index
+          - text: Overview of DevTools
+            url: /microsoft-edge/devtools-guide-chromium/index
+          - text: Change DevTools placement (Undock, Dock to bottom, Dock to left)
+            url: /microsoft-edge/devtools-guide-chromium/customize/placement
+          - text: Run commands in the Command Menu
+            url: /microsoft-edge/devtools-guide-chromium/command-menu/index
+          - text: Run JavaScript in the Console
+            url: /microsoft-edge/devtools-guide-chromium/console/console-javascript
+          - text: Keyboard shortcuts
+            url: /microsoft-edge/devtools-guide-chromium/shortcuts/index
+          - text: What's new in Microsoft Edge DevTools 
+            url: /microsoft-edge/devtools-guide-chromium/whats-new/whats-new
+          - text: Contact the Microsoft Edge DevTools team
+            url: /microsoft-edge/devtools-guide-chromium/contact
+
+# =============================================================================
+# Card
+  - title: Customizing DevTools
+    linkLists:
+      - linkListType: tutorial
```

Comment:
@codepo8 - 
The Customizing card should have multiple doc-type categories, unless these are all Tutorial articles.
The Debugging card should have multiple doc-type categories, unless these are all How-to articles.
The Emulation card should have multiple doc-type categories, unless these are all How-to articles.
The JavaScript card should have multiple doc-type categories, unless these are all Overview articles.

Reply:
I don't quite see the value of those, can we also omit them?

Reply:
The schema requires them.  I updated the Limits comments in this file:
# Number of cards in a landing page: 1-12
# Number of linkListType in a card: 1-6
# Number of links in a linkListType: 1-10